src/super_gradients/training/utils/detection_utils.py

```diff
@@ -701,12 +712,8 @@ def __call__(self, data) -> Tuple[torch.Tensor, torch.Tensor]:
         if self.random_resize_sizes is not None:
             data = self.random_resize(data)
 
-        batch = default_collate(data)
-        ims, targets = batch
-        targets = self._format_targets(targets)
-        ims = torch.moveaxis(ims, -1, 1).float()
-
-        return ims, targets
+        images_batch, labels_batch = list(zip(*data))
+        return self._format_images(images_batch), self._format_targets(labels_batch)
```

Comment:
Maybe we can call super() not not duplicate this part

Reply:
Good point, thanks! 91b0352